apps/consumer-client/pages/examples/add-pcd.tsx

```diff
@@ -62,11 +74,201 @@ export default function Page() {
         <button onClick={addWebAuthnPCD}>
           add a new webauthn credential to the passport
         </button>
+        <br />
+        <br />
+        <AddEthAddrPCDButton />
+        <br />
+        <br />
+        <AddEthGroupPCDButton />
       </ExampleContainer>
     </div>
   );
 }
 
+function AddEthAddrPCDButton() {
+  const [pcdStr] = usePassportPopupMessages();
+  const [isActive, setIsActive] = useState(false);
```

Comment:
The reason `isActive` is here is because both `AddEthAddrPCDButton` and `AddEthGroupPCDButton` are listening to the passport popup messages.

This is a simple way to disambiguate which one requested the message.